x/pylons/keeper/msg_server_trade_test.go

```diff
@@ -68,32 +18,33 @@ func (suite *IntegrationTestSuite) TestTradeMsgServerCreate1() {
 
 	enabled := true
 	params := banktypes.DefaultParams()
-	// app.BankKeeper.SetParams(ctx, params)
 
 	wctx := sdk.WrapSDKContext(ctx)
 	srv := keeper.NewMsgServerImpl(k)
 
 	creator := types.GenTestBech32FromString("creator")
 
 	fooCoin := sdk.NewCoin("testPylons", sdk.OneInt())
-	var items []types.Item
+	// bondCoin := sdk.NewCoin("testPylons1", sdk.OneInt())
```

Comment:
is this code necessary, or dead code?

Reply:
comments aren't code

Reply:
I have remove this line of code in https://github.com/Pylons-tech/pylons/pull/790/commits/7de5b33f5fcbc50325faaa6ca0a096ad79364181. Thank @mustafapylons and @faddat verymuch :+1: